src/styles/blocks/stars.scss

```
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) left no-repeat;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--0 :nth-child(0),
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background: url(../images/star-active.svg) left no-repeat;
```

can be simplify, there is no need to set `left no-repeat`
```suggestion
  background-image: url(../images/star-active.svg);
```